src/api/cvc4cpp.cpp

```diff
@@ -3793,6 +3801,8 @@ Term Solver::mkNegZero(uint32_t exp, uint32_t sig) const
 Term Solver::mkRoundingMode(RoundingMode rm) const
 {
   CVC4_API_SOLVER_TRY_CATCH_BEGIN;
+  CVC4_API_CHECK(Configuration::isBuiltWithSymFPU())
+      << "Expected CVC4 to be compiled with SymFPU support";
```

Comment:
Can you add this check also to:
https://github.com/CVC4/CVC4/blob/master/src/api/cvc4cpp.cpp#L3313
https://github.com/CVC4/CVC4/blob/master/src/api/cvc4cpp.cpp#L3278?
I noticed that it's missing there, too.